### PR TITLE
Unchecked the agents when you click outside

### LINKED ISSUE #2889: Selected agents are not unchecked when switching from Available agents to Current agents in the group in Manage agents option
| Wazuh | Elastic | Rev |
| ----- | ------- | --- |
| 4.1.0   | 7.10.0     | 4101-1 |

**Description**

Hello team,

When you are managing agents in `Manage agent` of a group menu, if you change from `Available agents` to `Current agents`, selected agents don't unmark and interface become confusing due to the lack information of which set of agents did you selected.

**Steps to reproduce**

- Go to `Management > Groups`
- Select a group
- Select `Manage agents`
- Select a set of agents from `Current agents` and then, select a set of agents from `Available agents`. `Add selected items` and r`emove selected items` options change but selected agents are not unmarked

**Video**

https://user-images.githubusercontent.com/11089305/106595055-463c9b00-6553-11eb-9754-522842b00ef9.mp4

**Additional context**
This happens in all browsers and system

### PR DESCRIPTION
**Pull request by pablomarga**
Hi team, this PR solved the problem that agents were not unchecked when switching from Available agents to Current agents in `Management > Groups`.
Closes #2889

### CODE REVIEW

File: public/components/management/groups/multiple-agent-selector.tsx [hunk 1]
```diff
@@ -498,6 +498,9 @@ export class MultipleAgentSelector extends Component {
                                 size='15'
                                 multiple
                                 onChange={(e) => {
+                                  document.getElementById("wzMultipleSelectorRight").childNodes.forEach(option =>  {
```
Comment (gabiwassan) [inline public/components/management/groups/multiple-agent-selector.tsx:501]: Can you put all of this code in a new method and maybe you can reuse the method with param wzMultipleSelectorRight/wzMultipleSelectorLeft

**Review by gabiwassan — APPROVED** [review]
LGTM

File: public/components/management/groups/multiple-agent-selector.tsx [hunk 2]
```diff
@@ -344,6 +344,19 @@ export class MultipleAgentSelector extends Component {
...
+        return document.getElementById("wzMultipleSelectorLeft").childNodes.forEach(option => {
+          option.selected = false
+        })
+      case 'right':
+        return document.getElementById("wzMultipleSelectorRight").childNodes.forEach(option => {
+          option.selected = false
+        })
+    }
+  }
+
```
Comment (Desvelao) [inline public/components/management/groups/multiple-agent-selector.tsx:359]: Why not use the container `id` as the param. You could simplify to:

```
unselectElementsOfSelectByID(containerID) {
    document.getElementById(containerID).selectedNodes.forEach(option => {
      option.selected = false
    })
  }
```

**Review by Desvelao — APPROVED** [review]
LGTM!

**Pull request merged**